css/deltagreen.css

```
@@ -1233,3 +1233,7 @@ body:not(.no-theme) #interface:not(.theme-light) .ui-control {
 body:not(.no-theme) #interface:not(.theme-light) #action-bar .slot.full:hover {
   border: 2px solid #215112;
 }
+
+.targetProficiencyModification {
```

Use kebab-case wherever possible with css/class names

```suggestion
.target-proficiency-mod {
```